lib/src/test/java/com/auth0/android/lock/errors/SignUpErrorMessageBuilderTest.java

```diff
@@ -27,7 +27,7 @@
 
     @Before
     public void setUp() {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
the previous one is marked as deprecated. In some classes, I had to change it to this new way for the tests to run successfully.